net/rpcbind/Makefile

```diff
@@ -0,0 +1,41 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=rpcbind
+PKG_VERSION:=0.2.4
+PKG_RELEASE:=1
+
+PKG_SOURCE_URL:=@SF/rpcbind
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_HASH:=074a9a530dc7c11e0d905aa59bcb0847c009313f02e98d3d798aa9568f414c66
+
+PKG_MAINTAINER:=Andy Walsh <andy.walsh44+github@gmail.com>
+PKG_LICENSE:=BSD-3-Clause
+
+PKG_FIXUP:=autoreconf
+PKG_REMOVE_FILES:=autogen.sh aclocal.m4
+PKG_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/rpcbind
+  SECTION:=net
+  CATEGORY:=Network
+  DEPENDS:=+libwrap +libtirpc
+  TITLE:=Universal addresses to RPC mapper 
+  URL:=http://libtirpc.sourceforge.net/
+endef
+
+CONFIGURE_ARGS += \
+	--enable-libwrap \
+	--with-rpcuser=root \
```

Comment:
I suggest you add libwrap as a configure option to avoid a dependency--I used libwrap on the nfs server, but not on the clients, and use rpcuser=rpc for a little bit of extra security, like the legacy portmap does.

Reply:
done